### PR TITLE
Hybrid and long read assemblies

### PR DESCRIPTION
**Pull request by mberacochea**
Assembly analysis fall in 3 categories now:
- assembly (good'ol assemblies)
- long_reads
- hybrid

The only change users should note on the website is going to be on the Analysis landing page. The instruments used for sequencing will be inferred from the Reads not the analysis for hybrids.

Tests will fail, they are dependent of the webkit / api to be merged first.

### CODE REVIEW

File: cypress/integration/genome.js [hunk 1]
```diff
@@ -23,11 +23,13 @@ describe('Genome page', () => {
                   .each(($row, idx) => {
                     $row.children('.column').each((_, el) => {
                         const $el = Cypress.$(el);
+                        const text = $el.text().replace(/\s+/g, ' ').replaceAll('\n', '');
```
Comment (gustavo-salazar) [inline cypress/integration/genome.js:26]: Just curious... isn't `\n` match for `/\s+/g` making the `replaceAll` redundant?

Comment (mberacochea) [reply cypress/integration/genome.js:26]: You are right.... that is a "trying to make the test pass" kind of error.

File: src/partials/experymentDetails.handlebars [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        {{#if model.runs}}
+            <div class="row">
+                {{#each model.runs}}
+                    {{#if @first }}
+                        <div class="column small-12 medium-4 large-4">
+                            Instruments:
+                        </div>
+                    {{/if}}
+                    <div data-cy="instrument" class="column small-12 medium-8 large-8 {{#unless @first}}small-offset-12 medium-offset-4 large-offset-4{{/unless}}">
+                        platform: {{this.instrument_platform}}, model: {{this.instrument_model}}
```
Comment (gustavo-salazar) [inline src/partials/experymentDetails.handlebars:29]: This render a bit weird, e.g:

| Experiment type:  |  assembly |
|-------------------|-----------|
|Instrument model: Illumina HiSeq 2000 | Instrument platform: ILLUMINA |

Sort of:
| KEY:  |  VALUE |
|-------------------|-----------|
|KEY: VALUE | KEY: VALUE |

**Review by gustavo-salazar — APPROVED** [review]
I'm happy with the changes, maybe check the formatting of the table as mentioned above.

**Review by SandyRogers — APPROVED** [review]
👏

File: src/partials/experymentDetails.handlebars (hunk 2 above)
Comment (mberacochea) [reply src/partials/experymentDetails.handlebars:29]: It should be, for hybrids

| Experiment type:  |  assembly |
|-------------------|-----------|
| Intruments:    | model: ILLUMINA, platform: Illumina HiSeq 2000 |
|     | model: PacBio , platform: RS II Instrument |

Comment (mberacochea) [reply src/partials/experymentDetails.handlebars:29]: It should be, for hybrids

| Experiment type:  |  assembly |
|-------------------|-----------|
| Intruments:    | model: ILLUMINA, platform: Illumina HiSeq 2000 |
|     | model: PacBio , platform: RS II Instrument |

I'll check this

**Pull request merged**